src/components/common/cards/card01.jsx

```diff
@@ -0,0 +1,86 @@
+import styled from "styled-components";
+import Card from "../../../assets/images/freezonecard.png";
+
```

Comment:
why you to copy same code for 3 card components?  make common card one and use it using map bro!